app/src/main/java/org/oppia/app/player/state/StateFragment.kt

```diff
@@ -47,6 +50,12 @@ class StateFragment : InjectableFragment(), InteractionAnswerReceiver {
 
   fun handleKeyboardAction() = stateFragmentPresenter.handleKeyboardAction()
 
+  override fun onPendingAnswerError(
+    pendingAnswerError: String?
```

Comment:
Change this to single line:

`override fun onPendingAnswerError(pendingAnswerError: String?) {`

Reply:
updated